php/class-fm-bylines-author.php

```diff
@@ -1,17 +1,35 @@
 <?php
 /**
  * Setup Author specific hooks using fieldmanager-bylines
+ *
+ * @package Fieldmanager_Bylines
  */
+
 if ( ! class_exists( 'FM_Bylines_Author' ) ) {
 
+	/**
+	 * Class FM_Bylines_Author
+	 */
 	class FM_Bylines_Author extends FM_Bylines_Post {
-
+		/**
+		 * The one true author object
+		 *
+		 * @var object
```

Comment:
You can make this more specific with `@var FM_Bylines_Author` which helps with IDE type checking.